tests/e2e/api/backups.e2e.spec.ts

```diff
@@ -164,6 +170,19 @@ describe(describeName('Backup E2E tests'), () => {
 
                 expect((await (await (app.getService<BackupService>(BackupService.name))).getAll()).length).equal(existingBackups.length + 1);
             });
+
+            it('should throw an exception if process is locked', async () => {
+
+                backupService.create()
```

Comment:
Test shouldn't finish until the backup is created. We should "wait" until `.create()` promise is resolved